test/helpers/DataConverterTests.scala

```diff
@@ -4,30 +4,33 @@ import scala.resources.BaseTestingClass
 
 class DataConverterTests extends BaseTestingClass{
 
+  val imageJPEG = "test/scala/resources/testImages/test.jpg"
+  val imagePNG = "test/scala/resources/testImages/test1.png"
+
   var dataConverter: DataConverter = _
 
   before {
     dataConverter = new DataConverter
   }
 
   "imageToBinaryData function" should "return the datatype Array[Byte]" in {
-    val filename = "C:/Users/Admin/IdeaProjects/ScalaFineArtApplication/public/images/favicon.png"
-    assert(dataConverter.imageToBinaryData(filename).isInstanceOf[Array[Byte]])
+    assert(dataConverter.imageToBinaryData(imageJPEG).isInstanceOf[Array[Byte]])
   }
 
   "imageToBinary function" should "be able to accept jpeg and png filetypes" in {
-    val png = "C:/Users/Admin/IdeaProjects/ScalaFineArtApplication/public/images/favicon.png"
-    val jpeg = "C:\\Users\\Admin\\IdeaProjects\\ScalaFineArtApplication\\test\\scala\\resources\\testImages\\test.jpg"
-    assert(dataConverter.imageToBinaryData(png).isInstanceOf[Array[Byte]])
-    assert(dataConverter.imageToBinaryData(jpeg).isInstanceOf[Array[Byte]])
+    assert(dataConverter.imageToBinaryData(imagePNG).isInstanceOf[Array[Byte]])
+    assert(dataConverter.imageToBinaryData(imageJPEG).isInstanceOf[Array[Byte]])
 
   }
 
   "binaryDataToImage" should "return an image file" in {
-    val image = "C:/Users/Admin/IdeaProjects/ScalaFineArtApplication/public/images/favicon.png"
-    val binData = dataConverter.imageToBinaryData(image)
+    val binDataJPEG = dataConverter.imageToBinaryData(imageJPEG)
+    val filenameJPEG = "test.jpg"
+    val binDataPNG = dataConverter.imageToBinaryData(imageJPEG)
+    val filenamePNG = "test1.png"
+    assert(dataConverter.binaryDataToImage(binDataJPEG, filenameJPEG) == true)
+    assert(dataConverter.binaryDataToImage(binDataPNG, filenamePNG) == true)
```

Comment:
`filenameJPEG` & `filenamePNG` should come from constants too also why store it in variable and use it once